packages/core/src/browser/hover-service.ts

```diff
@@ -124,6 +132,7 @@ export class HoverService {
             dispose: () => {
                 this.lastHidHover = Date.now();
                 host.classList.remove(updatedPosition);
+                host.classList.remove(...cssClasses ?? []);
```

Comment:
minor: I believe it makes more sense to wrap the call in an if statement rather than have an unecessary call to `remove([])`.